src/ImageCarouselReact/widget/ImageCarouselReact.tsx

```diff
@@ -0,0 +1,355 @@
+/*
+ ImageCarouselReact
+ ========================
+ 
+ @file      : ImageCarouselReact.js
+ @version   : 1.0.0
+ @author    : Akileng Isaac
+ @date      : Tue, 20 Sept 2016
+ @copyright : Flock of Birds 
+ @license   : Apache License V2.0
+ 
+ Documentation
+ ========================
+  ImageCarousel Widget displays an image carousel based on URLS 
+  stored in the domain model built with React-TypeScript . 
+ */
+
+import * as dojoDeclare from "dojo/_base/declare";
+// tslint:disable-next-line : no-unused-variable
+import * as React from "ImageCarouselReact/lib/react";
+import * as _WidgetBase from  "mxui/widget/_WidgetBase";
+import ReactDOM = require("ImageCarouselReact/lib/react-dom");
```

Comment:
Tried using - import * as ReactDOM from "ImageCarouselReact/lib/react-dom"; but the import fails for some reason. I have no clue why.